ExtraDry/Sample.Shared/Entitites/Region.cs

```diff
@@ -50,15 +59,17 @@ public class Region : TaxonomyEntity<Region>, ITaxonomyEntity, IValidatableObjec
     /// <remarks>
     /// Limited to 100 characters based on full names of countries which, in English, max at 59 characters per ISO.
     /// </remarks>
-    [Required, StringLength(100)]
+    [Required]
```

Comment:
I like to group like annotations together.  E.g. validation attributes with validation attributes, data attributes with data attributes.  Hence having Required and StringLength on the same line, they both are required to define the EF data type.

Reply:
I've put these back together. I didn't consciously separate them, it was a result of me looking at the required fields when creating an instance from the activator.